<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectIVClass</class>
 <widget class="QMainWindow" name="ProjectIVClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjectIV</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>921</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(42, 42, 63);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QPushButton" name="registerButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>310</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border: 2px solid rgb(37,39,48);
border-radius: 10px;
color: #FFF</string>
      </property>
      <property name="text">
       <string>Register Page</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>310</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid rgb(37,39,48);
border-radius: 10px;
color: #FFF
</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameLine">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>STXihei</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	
border: 2px solid rgb(18, 18, 27);
border-radius: 10px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;

	background-color: rgb(42, 42, 63);
}
QLineEdit:hover{
border: 2px solid rgb(42, 42, 63);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLine">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>230</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>STXihei</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
border: 2px solid rgb(18, 18, 27);
border-radius: 10px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;

	background-color: rgb(42, 42, 63);
}
QLineEdit:hover{
border: 2px solid rgb(42, 42, 63);
}</string>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="registerPage">
     <widget class="QPushButton" name="loginPageButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>290</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Login Page</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>220</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TEST</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="menuPage">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>TicTacToe</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(49, 104, 255);
border: 2px solid;

border-radius: 58px;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>50</y>
        <width>221</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string>ChatRoom</string>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>461</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Client 1: Hi                              16:42</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>560</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>540</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Send A Message</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>560</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>351</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>9</number>
       </property>
       <item row="2" column="2">
        <widget class="QPushButton" name="button22">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="button02">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="button11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="button00">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="button10">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="button12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="button20">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="button01">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="button21">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ProjectIV.qrc"/>
 </resources>
 <connections/>
</ui>
